acmwebsite/templates/profile_edit.xhtml

```diff
@@ -0,0 +1,19 @@
+<html py:strip=""
+  xmlns:py="http://genshi.edgewall.org/"
+  xmlns:xi="http://www.w3.org/2001/XInclude">
+
+  <py:extends href="master.xhtml" />
+
+  <head py:block="head" py:strip="True">
+      <title>Mines ACM - ${u}</title>
```

Comment:
Hard coded site name

Reply:
This seems to be a problem on all of the pages